OgreMain/src/OgreSceneManager.cpp

```diff
@@ -398,14 +398,14 @@ void SceneManager::_populateLightList(const Vector3& position, Real radius,
         std::stable_sort(destList.begin(), destList.end(), lightLess());
     }
 
-    // Now assign indexes in the list so they can be examined if needed
-    size_t lightIndex = 0;
-    for (LightList::iterator li = destList.begin(); li != destList.end(); ++li, ++lightIndex)
-    {
-        (*li)->_notifyIndexInFrame(lightIndex);
-    }
-
+    //Skip this, light indexes updated in findLightsAffectingFrustum
 
+    // Now assign indexes in the list so they can be examined if needed
+    //size_t lightIndex = 0;
+    //for (LightList::iterator li = destList.begin(); li != destList.end(); ++li, ++lightIndex)
```

Comment:
please do not leave commented out code behind. it does not help anyone

Reply:
ok, deleted